packages/core/phase2/exceptions/HO200106.test.ts

```diff
@@ -0,0 +1,62 @@
+import HO200106 from "./HO200106"
+import ExceptionCode from "bichard7-next-data-latest/dist/types/ExceptionCode"
+import errorPaths from "../../lib/exceptions/errorPaths"
+import generateAhoFromOffenceList from "../tests/fixtures/helpers/generateAhoFromOffenceList"
+import type { Offence } from "../../types/AnnotatedHearingOutcome"
+
+describe("HO200106", () => {
```

Comment:
This test suite does not cover all lines of code.
We need to add tests for when HO200106 function doesn't return an exception.

e.g.
`returns no exceptions when there is no fixed penalty`
`returns no exceptions when PNC adjudication exists `
etc